src/utils/getFilenameFromUrl.js

```diff
@@ -53,8 +53,14 @@ export default function getFilenameFromUrl(context, url) {
         filename = path.join(outputPath, querystring.unescape(pathname));
       }
 
-      let fsStats;
+      if (
+        !context.outputFileSystem.existsSync(filename) &&
+        options.historyApiFallback
```

Comment:
Change order, otherwise we will do fs check every time

Reply:
reverse order, really?

Reply:
```
function a() {
  console.log('a') 
  return false
}

function b() {
  console.log('b') 
  return true
}
```

if (a() && b()) { console.log("Yeah1") }
console.log("--------")
if (b() && a()) { console.log("Yeah2") }
```
a
--------
b
a
```

Reply:
`historyApiFallback` is false by default, but now we **always** do `!context.outputFileSystem.existsSync(filename)`. if `options.historyApiFallback` will be `false` we do not execute `!context.outputFileSystem.existsSync(filename)`

Reply:
Please read how `&&` and `||` works

Reply:
Sorry. It is my mistake. late evening...